lib/evss/base_service.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
+class EVSSErrors < Faraday::Response::Middleware
+  def on_complete(env)
+    case env[:status]
+    when 200
+      resp = env.body
+      if resp['success'] && !resp['success']
+        raise Exception, resp['messages']
```

Comment:
you can take a look at what we did in va_rx, those errors all inherit from the same base class

Eventually, the idea is to get those wrapped with the custom exception classes in va-api-common gem, which will allow us to do what we're doing in application_controller.rb#rescue_from

I think for now, just do what seems prudent or works well for you. I'm still working on pulling va_rx out of the gem. It's going to be tricky to build an abstract client object that everyone can use so for now it will just be a va_rx client, but hopefully as other clients emerge, we can abstract and DRY common components -- including error classes for json serialization, timeouts, etc